tests/unit/services/emailOtp.test.ts

```diff
@@ -0,0 +1,59 @@
+import { AxiosInstance } from 'axios';
+import { BaseError } from '../../../src/errors';
+import { EmailCodeSendReq, EmailCodeValidateReq } from '../../../src/generated';
+import { EmailOTP } from '../../../src/services';
+import Utils from '../../utils';
+
+describe('EmailOtp class', () => {
+  let axiosInstance: AxiosInstance;
+
+  beforeEach(() => {
+    axiosInstance = Utils.AxiosInstance();
+  });
+
+  it('should create EmailOTP instance with AxiosInstance', () => {
+    const emailOTP = new EmailOTP(axiosInstance);
+    expect(emailOTP).toBeInstanceOf(EmailOTP);
+  });
+
+  it('should send email code and return EmailCodeSendRsp', async () => {
+    const emailOTP = new EmailOTP(axiosInstance);
+    const req: EmailCodeSendReq = {
+      email: 'test@example.com',
```

Comment:
As this triggers real emails we should replace this email address with one of the integration-test email addresses also to avoid bounced emails.